### PR TITLE
Update KEB swagger with OSB schema

### PR DESCRIPTION
**Pull request by polskikiel**
**Description**

Changes proposed in this pull request:

- Updated swagger schema with OSB API endpoints. Additionally added tags, externalDocumentation and security.
- Updated the swagger-ui js bundle to the latest version: https://github.com/swagger-api/swagger-ui/releases/tag/v3.46.0

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/Dockerfile.keb [hunk 1]
```diff
@@ -22,4 +27,9 @@ COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certifica
 COPY --from=build /bin/kyma-env-broker /bin/kyma-env-broker
 COPY /files/swagger /swagger
 
+COPY --from=build /user/group /user/passwd /etc/
+USER appuser:appuser
+
+COPY --from=build --chown=appuser /swagger.yaml /swagger/swagger.yaml
```
Comment (polskikiel) [inline components/kyma-environment-broker/Dockerfile.keb:33]: added user to allow app write to file `swagger.yaml`

**Comment by polskikiel** [comment]
to use the oauth authorization from swagger UI you need to use that https://chrome.google.com/webstore/detail/allow-cors-access-control/lhobafahddgcelffkeicbaginigeejlf extension

File: docs/kyma-environment-broker/03-11-swagger.md [hunk 2]
```diff
@@ -19,7 +21,7 @@ Open the following website:
    https://$BROKER_URL/
    ```
 
-> **NOTE:** If you choose this option, you can't use the `Try it out` feature as the OAuth2 Swagger schema is not configured.
+With this option you need the browser [extension](https://chrome.google.com/webstore/detail/allow-cors-access-control/lhobafahddgcelffkeicbaginigeejlf) to use the `Try it out` feature.
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-11-swagger.md:24]:
```suggestion
To use the `Try it out` feature on Virtual Service, you need this [Chrome browser extension](https://chrome.google.com/webstore/detail/allow-cors-access-control/lhobafahddgcelffkeicbaginigeejlf).
```

File: docs/kyma-environment-broker/03-11-swagger.md [hunk 3]
```diff
@@ -9,6 +9,8 @@ The Swagger UI static files are copied from the [official source](https://github
 
 KEB uses a Swagger schema file that is mounted as a volume to the Pod. You can find that schema [here](https://github.com/kyma-project/control-plane/blob/main/resources/kcp/charts/kyma-environment-broker/files/swagger.yaml).
 
+You can use templates in the Swagger schema file to parametrize it.
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-11-swagger.md:12]:
```suggestion
You can use templates in the Swagger schema file to configure it.
```

Comment (klaudiagrz) [reply docs/kyma-environment-broker/03-11-swagger.md:12]: Please join this sentence with the previous one.

**Review by klaudiagrz — APPROVED** [review]

**Review by ksputo — DISMISSED** [review]
lgtm

**Review by kazydek — APPROVED** [review]
Previously reviewed and approved so just reapproving :)

**Review by piotrmiskiewicz — APPROVED** [review]

**Pull request merged**